sources/dataset.py

```diff
@@ -163,6 +139,73 @@ class (useful for semantic segmentation result printing)
                                      "aggregate": aggregate,
                                      "color": color}
 
+    def save(self, filename):
+        """Save dataset in a json file indicated by `filename`
+
+        Parameter
+        ---------
+        filename: object
```

Comment:
There is a whitespace between `filename`and the `:` (`nb_images` too).

Reply:
Done